Cargo.toml

```diff
@@ -70,6 +75,8 @@ cortex-m-log = { version = "0.7", features = ["log-integration"] }
 cfg-if = "0.1.10"
 mpu6050 = "0.1.4"
 bme680 = "0.6.0"
+sh1106 = { git = "https://github.com/techmccat/sh1106.git", branch = "hal-1", version = "0.5.0" }
```

Comment:
Is there any reason we are not using the regular 0.5.0 from crates.io?

Reply:
I was using my fork to test the i2c implementation for e-hal 1.
I also thought I hadn't included this example in the new set of changes but I must've re-added it by mistake at some point.